### DISCUSSION TITLE
how to solve this warning?

### DISCUSSION THREAD

**Discussion by aafnnp** [egoist/tsup#663] [Q&A]
![image](https://user-images.githubusercontent.com/12791216/175888108-223c25a8-9924-4df5-8f17-60bfd87ece1e.png)

**Comment by whoisryosuke** [comment]
Having this similar issue with importing assets like images, and then importing the module into a Vite app. 

Vite assumes images are located relatively, and `./` is relative to Vite root/public folder (not module `dist` folder).

I tried adding a loader to Base64, doesn't work either.

Not sure how to edit the `tsup` config to change the image URL - or get a Vite config that respects relative module imports.
